docs/aws-config.md

```diff
@@ -149,6 +149,119 @@ Specify the following values in the example files:
 
     * Replace `<YOUR_STORAGE_CLASS_NAME>` with `gp2`. This is AWS's default `StorageClass` name.
 
+## ALTERNATIVE: Setup permissions using kube2iam
+
+> This path assumes you have `kube2iam` already running in your Kubernetes cluster. If that is not the case, please install it first, following the docs here: https://github.com/jtblin/kube2iam
+
+[https://github.com/jtblin/kube2iam](Kube2iam) Kubernetes app allows managing AWS IAM permissions for pod via annotations rather than operating on API keys.
```

Comment:
The markdown order is incorrect for link text vs url - it needs to be `[kube2iam](https://github.com/jtblin/kube2iam)`. I'm also not sure if you're trying to say something like "kube2iam _is a_ Kubernetes app _that_ allows managing..."?

Reply:
Updated according to your comments.